src/domain/session/room/RoomViewModel.js

```diff
@@ -197,18 +197,80 @@ export class RoomViewModel extends ViewModel {
         }
     }
     
+    async _getMessageInformations (message) {
+        let msgtype = "m.text";
+        if (message.startsWith("/")) {
+            let command=message.substring(1).split(" ");
+            switch (command[0]) {
+                case "me":
+                   message = message.substring(4).trim();
```

Comment:
With the above suggestion, to get the rest of the message excluding the command, you can use:
```suggestion
                   message = arguments.join(" ").trim();
```
This is so you don't need to call `substring` with different argument for different commands.